src/RazorToolingTestApp.LibraryPackage/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": "1.0.0",
   "dependencies": {
-    "Microsoft.AspNetCore.Razor.Runtime": "1.1.0-*",
+    "Microsoft.AspNetCore.Razor.Runtime": "1.0.0",
```

Comment:
Should we have an alternate set of tests that verify this works with the new runtime?